app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,85 @@
+class RecipesController < ApplicationController
+  before_action :set_recipe, only: %i[show edit update destroy]
+  skip_before_action :authenticate_user!, only: %i[public_recipes show]
+
+  # GET /recipes or /recipes.json
+  def index
+    # logged in user recipes
+    @user_test = current_user
+    @recipes = Recipe.where(user: @user_test)
+  end
+
+  def public_recipes
+    @recipes = Recipe.includes(:recipe_foods, :foods).all
```

Comment:
- [ ] As the previous reviewer suggested kindly the public recipes should be placed where they belong. If I try to see current_user's created recipes are shown inside Public Recipes even if they are not marked as public. Also, I tried signing up for another user but in their list, it's only showing public good job for that. However, you need to display it only publics in public. Kindly complete the requirement as suggested. Thank you!

Reply:
Thanks for the review @Rustamxon7. Already change line 13 to select only public recipes.